src/app/utils/chart-utils.ts

```diff
@@ -69,3 +69,38 @@ export const filterHourlyActiveAccounts = (
 ): ActiveAccounts[] | undefined => {
   return windows?.filter((value, index) => index % 12 === 0)
 }
+
+export const sumBucketsByChucks = <T>(
+  buckets: T[] | undefined,
+  sumKey: keyof T,
+  dateKey: keyof T,
+  numOfChunks: number,
+) => {
+  if (!buckets) {
+    return []
+  }
+
+  const chunkArray = buckets.reduce(
+    (accArray, item, index) => {
+      const chunkIndex = Math.floor(index / numOfChunks)
```

Comment:
shouldn't we group by hour, something like isSameHour ?

Reply:
I was looking into active accounts chart, which has the "offset" as well. It would make more sense. Should I add it to Active accounts chart as well?

Reply:
It's not the same. In active accounts we show every 12th item and we do not group/sum them as windows overlaps for 5 minutes buckets.

Reply:
Right, after talking with Don, summing by hour makes the most sense.